src/psyclone/tests/dynamo0p3_basis_test.py

```diff
@@ -54,15 +54,16 @@
 CODE = '''
 module testkern_eval
   type, extends(kernel_type) :: testkern_eval_type
-    type(arg_type) :: meta_args(2) = (/                                  &
-         arg_type(GH_FIELD,   GH_INC,  W0),                              &
-         arg_type(GH_FIELD,   GH_READ, W1)                               &
+    type(arg_type) :: meta_args(2) = (/       &
+         arg_type(GH_FIELD,   GH_INC,  W0),   &
+         arg_type(GH_FIELD,   GH_READ, W1)    &
          /)
-    type(func_type) :: meta_funcs(2) = (/                                &
-         func_type(W0, GH_BASIS),                                        &
-         func_type(W1, GH_DIFF_BASIS)                                    &
+    type(func_type) :: meta_funcs(2) = (/     &
+         func_type(W0, GH_BASIS),             &
+         func_type(W1, GH_DIFF_BASIS)         &
          /)
     integer, parameter :: gh_shape = gh_evaluator
+    integer, parameter :: gh_evaluator_targets(2) = [W0, W1]
```

Comment:
Well spotted. I've made the corrections. As this code is written into the file (i.e. not generated by PSyclone), I don't think a test that it compiles is required. Also, it would be a pain to do so because `string_compiles` only works for code that has no external dependencies and this meta-data depends on LFRic modules to define certain types.